plugins/blip/basics/logic/datamunger.js

```diff
@@ -28,9 +28,9 @@ var constants = require('./constants');
 var basicsActions = require('./actions');
 var togglableState = require('../TogglableState');
 
-module.exports = function(bgClasses) {
+module.exports = function(bgClasses, bgUnits = 'mg/dL') {
```

Comment:
`mg/dL` is used so many places it could just be another constant possibly ... i know this is legacy code though

Reply:
I thought about moving to constants here (and did in the other repos), but decided against it after starting the process, given it's legacy code, as you mention.